scripts/release-asm/common.sh

```diff
@@ -170,10 +211,23 @@ write_and_upload() {
     upload "${SCRIPT_NAME}" "${FILE_NAME}" "${BUCKET_PATH}/${SCRIPT_NAME}" "${BUCKET_URL}/${BUCKET_PATH}/${SCRIPT_NAME}"
   fi
 
+  if [[ "${BRANCH_NAME}" =~ "release" ]]; then
+    append_version "${VERSION}" "${FILE_NAME}"
+  elif is_proper_tag "${VERSION}"; then
+    append_version "${VERSION}" "${FILE_NAME}"
+  fi
```

Comment:
Could this just be

```
if is_proper_tag "${VERSION}" || [[ "${BRANCH_NAME}" =~ "release" ]]; then
```

Reply:
I spent at least 15 minutes trying to write ``` if [[ "${BRANCH_NAME}" =~ "release" || is_proper_tag "${VERSION}"  ]]; then``` and failed to make the shellchecker happy.